vllm/envs.py

```diff
@@ -499,6 +500,14 @@ def maybe_convert_int(value: Optional[str]) -> Optional[int]:
     (os.environ.get("VLLM_ALLOW_RUNTIME_LORA_UPDATING", "0").strip().lower() in
      ("1", "true")),
 
+    # If set, vllm will look in this directory when it receives
+    # a request for a lora adapter it doesn't recognize. Only
+    # available if VLLM_ALLOW_RUNTIME_LORA_UPDATING is set
+    "VLLM_ADAPTER_DIR":
```

Comment:
I have a slight preference for incorporating `LORA` in the name here to remove any ambiguity about the word `ADAPTER`

Reply:
hmm, i kept it generic in case we wanted to use this in the future for other cached stuff, backwards compatability and all that. i can change it